UI/xdg-data/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+install(FILES com.obsproject.Studio.desktop DESTINATION ${CMAKE_INSTALL_FULL_DATAROOTDIR}/applications)
+install(FILES com.obsproject.Studio.appdata.xml DESTINATION ${CMAKE_INSTALL_FULL_DATAROOTDIR}/metainfo)
+install(FILES ../forms/images/obs.png DESTINATION ${CMAKE_INSTALL_FULL_DATAROOTDIR}/icons/hicolor/256x256/apps)
+
```

Comment:
Style:  While files must end in a newline character, they don't need an extra blank newline.  You can remove this line.

```suggestion
```